doc/ldap-demo.md

```diff
@@ -0,0 +1,194 @@
+# LDAP Demo
+
+Run a local ldap server, use docker for speed
```

Comment:
heh, my initial testing was with a docker ldap server, forgot to clear this out of the instructions when I switched to installing one via apt packages.